### ISSUE TITLE
Optional request data on io.socket.get not getting appended to url

### ISSUE THREAD

**Issue by bgulotta** [balderdashy/sails.io.js#54]
![screen shot 2015-05-22 at 7 56 14 am](https://cloud.githubusercontent.com/assets/3649271/7770853/3b73c290-0058-11e5-97df-9b6e0303d663.png)

The documentation states the optional data field on an io.socket.get request should be getting appended to the url. This currently doesn't appear to be working.

**Comment by barroudjo** [comment]
I reported a similar issue to the sails project, but I only had it when querying custom methods in my controllers.

https://github.com/balderdashy/sails/issues/3052

**Comment by sailsbot** [comment]
Thanks for posting, @bgulotta.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
